hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2400,8 +2400,14 @@
     <name>hdds.tracing.enabled</name>
     <value>true</value>
     <tag>OZONE, HDDS</tag>
+    <description>If enabled, tracing information is sent to tracing server.</description>
+   </property>
+  <property>
+    <name>ozone.s3g.address</name>
+    <value/>
+    <tag>OZONE, MANAGEMENT</tag>
     <description>
-      If enabled, tracing information is sent to tracing server.
+      The address of the Ozone S3G gateway.n.
```

Comment:
typo in end?

Reply:
Thanks. Removed the modification of ozone-default.xml as I switched to use the annotation based configs.